js/constants/message.js

```diff
@@ -0,0 +1,12 @@
+export const MESSAGE = Object.freeze({
+  ERROR: {
+    EMPTY_EMAIL: '이메일을 입력해주세요.',
+    REGEX_EMAIL: '올바른 이메일 주소가 아닙니다.',
```

Comment:
REGEX_EMAIL은 어떤 의미인지 당연히 추측할 수는 있는데요~
메시지 자체가 어떤 의미를 담고 있는지 표현해주는 것도 괜찮을 것 같아요!
```suggestion
    INVALID_EMAIL: '올바른 이메일 주소가 아닙니다.',
```